depmapomics/copynumbers.py

```diff
@@ -467,10 +469,10 @@ def _CCLEPostProcessing(
     )
     wgssegments, genecn, wgsfailed = postProcess(
         wgsrefworkspace,
-        AllSamplesetName if AllSamplesetName else samplesetname,
+        setEntity="sample_set",
+        sampleset=AllSamplesetName if AllSamplesetName else samplesetname,
         todrop=todropwgs,
         save_output=folder,
-        segmentsthresh=2000,
```

Comment:
I think in the end we decided to keep the threshold for WGS, didn't we?

Reply:
I don't remember this conversation but maybe I dropped it accidentally. I'll add it back